DscResources/MSFT_xMySqlServer/MSFT_xMySqlServer.schema.mof

```diff
@@ -2,19 +2,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlServer")] 
 class MSFT_xMySqlServer : OMI_BaseResource
 {
-  [Key, Description("Provides the service name to use during setup of MySQL")]
-  String ServiceName;
-  
-  [Write,
-  ValueMap{"Present", "Absent"},Values{"Present", "Absent"},
-  Description("Ensure resource is present or absent")]
-  String Ensure;
-  
-  [required,
-  Write,
-  EmbeddedInstance("MSFT_Credential"),
-  Description("The root credential that is used to install mySql server.")]
-  String RootPassword;
+    [Key, Description("mySql Version Number")] String MySqlVersion;
```

Comment:
It is not about running different versions of the MySQL as a service that use the default service name, but about installing multiple instances of the same MySQL version on the machine. That is why service name was the key in the original implementation.  Ideally speaking the key should have been serviceName and port pair .